ecommerce/shopper_experience/payment_acquirer.rst

```diff
@@ -1,78 +1,80 @@
-   payment status, when they are redirected to Odoo after the transaction.
-   To edit such messages, go to the *Messages* tab of the payment
+   payment status whenever they are redirected to Odoo after the transaction.
+   To edit these messages, go to the *Messages* tab of the payment
    method.
 
+Automatically generate invoices at order
+========================================
 
-Auto-validate invoices at order
-===============================
-
-When the order is confirmed you can also have an invoice automatically issued
-and paid. This fully-automated made for businesses that invoice 
+When the order is confirmed, you can also have an invoice automatically issued
+and paid. This fully-automated feature is designed for businesses that invoice 
 orders straight on.
 
-.. image:: media/payment_invoice.png
+To do this, go to the *Invoicing* section of the *Settings* in the Website application,
+and select *Invoice What Is Ordered* and then select *Automatic Invoices*, which will
+generate an invoice automatically when the online payment is confirmed.
+
+.. image:: media/automatic-invoice.png
     :align: center
 
 .. note::
-   If you choose this mode you are requested to select a payment journal in order to record payments
-   in your books. This payment is automatically reconcilied with the invoice, marking it as paid.
-   Select your **bank account** if you get paid immediately on your bank account. If you don't you
-   can create a specific journal for the payment acquirer (type = Bank). That way, you can track
+   If you choose this mode, you are requested to select a payment journal, in order to record 
+   payments in your books. This payment is automatically reconcilied with the invoice, marking it as 
+   paid. Select your **bank account** if you get paid immediately on your bank account. If you don't 
+   you can create a specific journal for the payment acquirer (type = Bank). That way, you can track
    online payments in an intermediary account of your books until you get paid into your bank
    account (see :doc:`../../general/payment_acquirers/payment_acquirers`).
 
+Capture payment after the delivery
+==================================
+With this mode (which is only available with Authorize.net), the order is confirmed, 
+but the amount is kept on hold. Once the delivery processed, you can capture the payment from Odoo.
 
-Capture the payment after the delivery
-======================================
-With this mode, the order is confirmed but the amount is kept on hold. 
-Once the delivery processed, you can capture the payment from Odoo.
-This mode is only available with Authorize.net.
-
-.. image:: media/payment_capture_mode.png
+.. image:: media/capture-manually.png
     :align: center
 
-To capture the payment, open the transaction from the order.
-Then click *Capture Transaction*.
+To capture the payment, open the transaction from the order in *Payment Transactions*.
```

Comment:
This is not in the hidden payment transactions menu anymore.
You can read more about it here https://www.odoo.com/documentation/user/14.0/general/payment_acquirers/authorize.html#capture-the-payment-after-the-delivery

Reply:
Great point! For the sake of efficiency, I'll simply add a quick link to this documentation, so as to avoid any unnecessary re-explaining. If this is the incorrect move, however, feel free to let me know how I should go about fixing this issue. Thanks again for all your continued patience and feedback during this all-too-important learning process. I sincerely appreciate it.